pandas/core/strings/base.py

```diff
@@ -0,0 +1,225 @@
+import abc
+from typing import Pattern, Union
+
+import numpy as np
+
+from pandas._typing import Scalar
+
+
+class BaseStringArrayMethods(abc.ABC):
+    """
+    Base class for extension arrays implementing string methods.
+
+    This is our ExtensionArrays can override the implementation of
```

Comment:
"our" -> "so"?

Reply:
Yeah. Probably not worth re-running CI for though. I'll fix it if anything else comes up.

Reply:
sounds good